<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>return_roll</class>
 <widget class="QDialog" name="return_roll">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Επιστροφές ρολών</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Καταχώρηση</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>301</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineScan">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Γίνεται ενημέρωση. Παρακαλώ περιμένετε</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
